### ISSUE TITLE
"TypeError: Cannot read properties of null (reading 'includes')" or "TypeError: o is null" error when selecting a project variable type 'Username Password Account'

### ISSUE THREAD

**Issue by Clare-Octopus** [OctopusDeploy/Issues#8544]
### Version

v2024.1.4490-hotfix.5037

### Latest Version

I could reproduce the problem in the latest build

### What happened?

When trying to insert a Project Variable of type `Username Password Account` an error is thrown (different errors for different browsers):

**Chrome Latest:**

`An unexpected error occurred in Octopus v2024.1.4490-hotfix.5037: "TypeError: Cannot read properties of null (reading 'includes')"`

**Firefox Latest:**

`An unexpected error occurred in Octopus v2024.1.4490-hotfix.5037: "TypeError: o is null"`

I have tried opening the editor and setting the `Username Password Account` option from there but it gives the same error. I then tried putting values in for the Variable name and also the value and then changing the option to `Username Password Account` but it again throws the error.

### Reproduction

1. Open any project and go to the Variables Tab.
2. Try changing the variable value type to `Username Password Account`.
3. See the error message.

### Error and Stacktrace

```shell
TypeError: Cannot read properties of null (reading 'includes')
    at https://octopus-operations.octopus.app/app-main.7f9a65591ecb9f9f6ae4.hashedasset.js:2:167026
    at Array.some (<anonymous>)
    at https://octopus-operations.octopus.app/app-main.7f9a65591ecb9f9f6ae4.hashedasset.js:2:167015
    at Array.filter (<anonymous>)
    at tu (https://octopus-operations.octopus.app/app-main.7f9a65591ecb9f9f6ae4.hashedasset.js:2:166996)
    at Qi (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1905846)
    at ks (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1951892)
    at xl (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1944652)
    at wl (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1944577)
    at hl (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1941600)
Captured By
 Error
    at Object.error (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:3018037)
    at Bn.componentDidCatch (https://octopus-operations.octopus.app/app-main.7f9a65591ecb9f9f6ae4.hashedasset.js:2:38883)
    at i.componentDidCatch.n.callback (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1937376)
    at gi (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1896492)
    at us (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1931483)
    at Ol (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1948994)
    at t.unstable_runWithPriority (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:2547427)
    at qo (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1892964)
    at _l (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1945653)
    at hl (https://octopus-operations.octopus.app/main.880361f6e4008d7ac716.hashedasset.js:2:1941800)
```

### More Information

Initial Ticket (Internal) - https://octopus.zendesk.com/agent/tickets/166418
SEQ Error from Octopus-Operations (Internal) - https://customerinstance-seq.octopushq.com/#/events/?filter=@Id%20%3D%20%27event-bc7b84400b6d08dcaa1c89d104000000%27&showExpanded&permalinkId=permalink-4435

### Workaround

No workaround at the moment. You should be able to create a Sensitive variable instead for the moment and type the username of the account as the Variable name and the password as the Variable value. With it being a sensitive variable type Octopus will hask out the password value and you can scope it to environments etc.

**Comment by Clare-Octopus** [comment]
Duplicate of https://github.com/OctopusDeploy/Issues/issues/8530 which is fixed in 2024.1.6217. This will start rolling out to Cloud customers soon but if you are running into this and would like to be upgraded right away please contact Support - support@octopus.com and we will schedule an upgrade of your cloud instance.

**Comment by Clare-Octopus** [comment]
This issue has now been resolved, please see my previous comment above.
